core/primitives/src/crypto.rs

```diff
@@ -897,6 +897,8 @@ pub mod key_types {
 	pub const AURA: KeyTypeId = KeyTypeId(*b"aura");
 	/// Key type for ImOnline module, built-in.
 	pub const IM_ONLINE: KeyTypeId = KeyTypeId(*b"imon");
+	/// Key type for AuthorityDiscovery module, built-in.
+	pub const AUTHORITY_DISCOVERY: KeyTypeId = KeyTypeId(*b"audi");
```

Comment:
![image](https://user-images.githubusercontent.com/5718007/67929683-2c790f00-fbbe-11e9-836b-213ce7947261.png)

Reply:
... No, *listen* in Latin @bkchr :P

Reply:
Okay :P